### PR TITLE
fix(celery): remove unused queues and add a LONG_RUNNING queue

### PR DESCRIPTION
**Pull request by aspicer**
This can't be landed without helm changes seen here: https://github.com/PostHog/charts/pull/1301
## Problem

because of prefetching ( each worker prefetches 4 tasks to run at a time from the queue ), a long running query can block the other tasks in that workers prefetch queue.

## Changes

remove 2 unused queues

create a new "long_running" queue and move two known long running queries to it

## Does this work well for both Cloud and self-hosted?

Does self hosted need any sort of configuration changes to support new queues?

## How did you test this code?

It hasn't been tested. How should it be?

### CODE REVIEW

File: posthog/tasks/utils.py [hunk 1]
```diff
@@ -29,8 +29,7 @@ class CeleryQueue(Enum):
     DEFAULT = "celery"
     STATS = "stats"
     EMAIL = "email"
-    INSIGHT_EXPORT = "insight_export"
-    INSIGHT_REFRESH = "insight_refresh"
+    LONG_RUNNING = "long_running"  # any task that has a good chance of taking more than a few seconds should go here
```
Comment (benjackwhite) [inline posthog/tasks/utils.py:32]: Bear in mind that with this change any tasks queued in the old queues wont get consumed (at least if you also rename it in the chart)

Comment (aspicer) [reply posthog/tasks/utils.py:32]: I don't think that those queues are used at all anymore - I can't find any (other) references to them in the code.

File: posthog/tasks/utils.py [hunk 2]
```diff
@@ -29,8 +29,7 @@ class CeleryQueue(Enum):
     DEFAULT = "celery"
     STATS = "stats"
     EMAIL = "email"
-    INSIGHT_EXPORT = "insight_export"
-    INSIGHT_REFRESH = "insight_refresh"
+    LONG_RUNNING = "longrunning"  # any task that has a good chance of taking more than a few seconds should go here
```
Comment (webjunkie) [inline posthog/tasks/utils.py:32]:
```suggestion
    LONG_RUNNING = "long_running"  # any task that has a good chance of taking more than a few seconds should go here
```

Right? 🙈 

Maybe we should make all worker names != any queue names to reduce confusion?

**Review by webjunkie — APPROVED** [review]
Approving already. But check possible name confusion.

File: posthog/tasks/utils.py (hunk 2 above)
Comment (aspicer) [reply posthog/tasks/utils.py]: Lifesaver!

Yes, I think that makes sense in the future, although it sounds like an annoying change to make now.

Add new queues names in chart, deploy, change them here, deploy, remove old ones in chart? Will consider going forward.

Comment (webjunkie) [reply posthog/tasks/utils.py]: >it sounds like an annoying change to make now.

Agreed... that thought came slightly too late.

**Pull request merged**